libs/blocks/review/components/review/Ratings.js

```diff
@@ -87,7 +90,7 @@ const Ratings = ({
   };
 
   const onBlur = (ev) => {
-    if (ev.relatedTarget === null || ev.relatedTarget.nodeName !== 'INPUT') {
+    if (ev.relatedTarget === null || ev?.relatedTarget?.nodeName !== 'INPUT') {
```

Comment:
Looks like you are checking presence of `ev` (`ev?.relatedTarget?.nodeName`). But if `ev` is undefined, then you will get an exception while doing `ev.relatedTarget`